model/project_ref.go

```diff
@@ -2032,6 +2032,14 @@ func SaveProjectPageForSection(projectId string, p *ProjectRef, section ProjectP
 		if catcher.HasErrors() {
 			return false, errors.Wrapf(catcher.Resolve(), "validating external links")
 		}
+		var pRef *ProjectRef
+		pRef, err = FindBranchProjectRef(projectId)
+		if err != nil {
+			return false, errors.Wrapf(err, "error getting project")
+		}
```

Comment:
`pRef == nil` should be a second error case that we check right after this err check